crates/swc/tests/tsc-references/emitClassDeclarationWithThisKeywordInES6.2.minified.js

```diff
@@ -1,16 +1 @@
 //// [emitClassDeclarationWithThisKeywordInES6.ts]
-class B {
```

Comment:
I feel like this renders these test cases useless and they should better run with DCE disabled for top level symbols

Reply:
Will do it tomorrow, thanks!